flow/SystemMonitor.cpp

```diff
@@ -42,6 +42,12 @@ void systemMonitor() {
 	customSystemMonitor("ProcessMetrics", &statState, true);
 }
 
+void systemMonitorNetworkBusyness() {
```

Comment:
I don't think you need to use system monitor at all for this. It looks like you can remove all of that code and it will still work.

Reply:
I moved everything into one function but i think keeping it in SysteMonitor makes sense since the code is very similar to the other functions